rust/unicode/src/tables.rs

```diff
@@ -1497,15 +1498,15 @@ pub const LINEBREAK_STATE_MACHINE: [u8; 3827] = [
     128, 129, 130, 131, 132, 133, 6, 135, 8, 137, 10, 11, 140, 141, 142, 143,
     16, 17, 146, 147, 148, 149, 150, 151, 152, 153, 26, 27, 28, 29, 30, 159,
     160, 161, 162, 163, 36, 165, 166, 167, 168, 169, 170,
-    // state 86: SP+ ZWJ
+    // state 86: SP+ EB
     128, 129, 130, 131, 132, 133, 6, 135, 8, 137, 10, 11, 140, 141, 142, 143,
     16, 17, 146, 147, 148, 149, 150, 151, 152, 153, 26, 27, 28, 29, 30, 159,
     160, 161, 162, 163, 36, 165, 166, 167, 168, 169, 170,
-    // state 87: SP+ EB
+    // state 87: SP+ EM
     128, 129, 130, 131, 132, 133, 6, 135, 8, 137, 10, 11, 140, 141, 142, 143,
     16, 17, 146, 147, 148, 149, 150, 151, 152, 153, 26, 27, 28, 29, 30, 159,
     160, 161, 162, 163, 36, 165, 166, 167, 168, 169, 170,
-    // state 88: SP+ EM
+    // state 88: SP+ ZWJ
```

Comment:
can you summarize what the table changes represent?

Reply:
Yes, when I verified that the numeric values were the same as ICU ULineBreak, I found that I had gotten the order wrong, so this just renumbers those properties. The table has the same semantics, just permuted a bit.